app.js

```diff
@@ -1,32 +1,27 @@
 const express = require('express');
+const { join } = require('path');
 const { AsyncLocalStorage } = require('async_hooks');
-const { v4: uuid } = require('uuid');
 const { PORT, ENV } = require('./config');
-const logger = require('./logger');
 
+const asyncStorage = new AsyncLocalStorage();
 const Storage = require('./storage.js');
 const eventStorage = new Storage();
-const asyncStorage = new AsyncLocalStorage();
 
 module.exports = { asyncStorage, eventStorage };
 
-const router = require('./routs/events');
 
-
-const requestIdMiddleware = (req, res, next) => {
-  asyncStorage.run({
-    requestId: uuid(),
-    userId: uuid()
-  }, () => {
-    next();
-  });
-};
+const logger = require('./logger');
+const router = require(join(__dirname, 'routs', 'events.js'));
+const requestIdMiddleware = require(join(__dirname, 'middlewares', 'requestTracker.js'));
```

Comment:
you can require just with string, join for require is an overkill )) like for unix system it is ok